gwak/train/dataloader.py

```diff
@@ -881,9 +870,15 @@ def inject(self, batch, waveforms, output_snrs = False):
             if torch.any(torch.isnan(waveforms)):
                 self._logger.info('centered waveforms fucked')
 
-            rescaled_waveforms, snr_factor = self.rescaler.forward(
+            snr_factor = self.snr_prior.rsample((waveforms.shape[0],)).to(waveforms.device)
+            with h5py.File("/home/hongyin.chen/gwak-check_01.h5", "w") as f:
```

Comment:
i think this and the next line have to be removed! they are breaking the code and are a test anyway :)